src/ToJSON/parsing/parsing.ts

```diff
@@ -36,10 +42,23 @@ export function ParseText(text?: string, parsingOptions?: string, invokeProgress
   let lineNumber = 1;
   let lines = split(text, '\n');
   let yamlOptions: string | object | undefined = {};
+  let conversionYamlOptions: string | object | undefined = {};
+
+  let ignoreMaxLineLength = false;
 
   try {
     yamlOptions = yaml.safeLoad(parsingOptions);
     SetParsingOptions(yamlOptions);
+
+    conversionYamlOptions = yaml.safeLoad(conversionOptions ?? '');
+
+    if (conversionYamlOptions) {
+      const yamlConfig = JSON.parse(JSON.stringify(conversionYamlOptions)).Options;
+      if (yamlConfig) {
+        const configOptions: any = Object.assign({}, ...yamlConfig);
+        ignoreMaxLineLength = configOptions.IgnoreMaxLineLength.toString() === 'true';
+      }
+    }
```

Comment:
```suggestion
    const conversionYamlOptions = (yaml.safeLoad(conversionOptions ?? '') as any as Options) ?? new Options();
    SetValidationOptions(conversionYamlOptions.Validation ?? new ValidationOptions());
```
Why so complicated?!